<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsForm</class>
 <widget class="QWidget" name="SettingsForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>517</width>
    <height>469</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>Radio</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Tone</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Shaping</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>73</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="10" column="1">
        <widget class="QLabel" name="versionLabel">
         <property name="text">
          <string>TextLabel</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>CW-R</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QLabel" name="gainLabel">
         <property name="text">
          <string>1</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_21">
         <property name="text">
          <string>FFT Window</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QSpinBox" name="qskSpinBox">
           <property name="maximum">
            <number>999</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_23">
           <property name="text">
            <string>milliseconds</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="1">
        <widget class="QCheckBox" name="cwrCheckBox">
         <property name="text">
          <string> Receive on upper sideband.</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_19">
         <property name="text">
          <string>IQ Gain</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QSlider" name="fftFilterSlider">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>FFT Filter</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>IQ Phase</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QLabel" name="phaseLabel">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QDoubleSpinBox" name="shapeSpinBox">
           <property name="decimals">
            <number>1</number>
           </property>
           <property name="maximum">
            <double>6.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>milliseconds</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="windowComboBox">
         <item>
          <property name="text">
           <string>Polyphase</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Hann</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Rectangular</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_22">
         <property name="text">
          <string>Break-in</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QSpinBox" name="rxToneSpinBox">
           <property name="minimum">
            <number>250</number>
           </property>
           <property name="maximum">
            <number>2500</number>
           </property>
           <property name="value">
            <number>2500</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>Hz</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="5" column="1">
        <widget class="QComboBox" name="colorsComboBox">
         <item>
          <property name="text">
           <string>Winrad</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Linrad</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Grayscale</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Spectran</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Spectran Extended</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Horne</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>FFT Colors</string>
         </property>
        </widget>
       </item>
      </layout>
      <zorder>label_9</zorder>
      <zorder>label_13</zorder>
      <zorder>label_11</zorder>
      <zorder>cwrCheckBox</zorder>
      <zorder>label_21</zorder>
      <zorder>windowComboBox</zorder>
      <zorder>label</zorder>
      <zorder>colorsComboBox</zorder>
      <zorder>label_17</zorder>
      <zorder>fftFilterSlider</zorder>
      <zorder>label_14</zorder>
      <zorder>phaseLabel</zorder>
      <zorder>label_19</zorder>
      <zorder>gainLabel</zorder>
      <zorder>versionLabel</zorder>
      <zorder>label_22</zorder>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Keyer</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_18">
         <property name="text">
          <string>Tone</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QCheckBox" name="keyToneCheckBox">
         <property name="text">
          <string> Use same tone as receiver.</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QSpinBox" name="keyToneSpinBox">
           <property name="minimum">
            <number>250</number>
           </property>
           <property name="maximum">
            <number>2500</number>
           </property>
           <property name="value">
            <number>2500</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_20">
           <property name="text">
            <string>Hz</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Volume</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSlider" name="keyVolumeSlider">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Weight</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QSlider" name="keyWeightSlider">
         <property name="minimum">
          <number>40</number>
         </property>
         <property name="maximum">
          <number>60</number>
         </property>
         <property name="pageStep">
          <number>5</number>
         </property>
         <property name="value">
          <number>50</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksBothSides</enum>
         </property>
         <property name="tickInterval">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Mode</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="keyModeComboBox">
         <item>
          <property name="text">
           <string>Iambic</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Ultimatic</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Bug</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Memory</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QComboBox" name="keyMemoryComboBox">
         <item>
          <property name="text">
           <string>None</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Dash</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Dot</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Both</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Spacing</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QComboBox" name="keySpacingComboBox">
         <item>
          <property name="text">
           <string>None</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Element</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Character</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Word</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Reverse</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QCheckBox" name="keyReverseCheckBox">
         <property name="text">
          <string> Swap dot and dash.</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>88</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
      <zorder>label_18</zorder>
      <zorder>label_4</zorder>
      <zorder>keyVolumeSlider</zorder>
      <zorder>label_7</zorder>
      <zorder>keyWeightSlider</zorder>
      <zorder>label_6</zorder>
      <zorder>keyModeComboBox</zorder>
      <zorder>label_8</zorder>
      <zorder>keyMemoryComboBox</zorder>
      <zorder>label_5</zorder>
      <zorder>keySpacingComboBox</zorder>
      <zorder>label_10</zorder>
      <zorder>keyReverseCheckBox</zorder>
      <zorder>keyToneCheckBox</zorder>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>TX</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="txImageBox">
         <property name="title">
          <string>Image: 12.345.678 Hz</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_11">
          <item row="0" column="0">
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>Tune another receiver to the image frequency then adjust sliders to minimize transmitted image.</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="phaseBox">
         <property name="title">
          <string>Phase: 0</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_13">
          <item row="1" column="0">
           <widget class="QSlider" name="phaseFineSlider">
            <property name="minimum">
             <number>-1000</number>
            </property>
            <property name="maximum">
             <number>1000</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="tickPosition">
             <enum>QSlider::TicksBelow</enum>
            </property>
            <property name="tickInterval">
             <number>1000</number>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QSlider" name="phaseCoarseSlider">
            <property name="minimum">
             <number>-1000</number>
            </property>
            <property name="maximum">
             <number>1000</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="tickPosition">
             <enum>QSlider::TicksAbove</enum>
            </property>
            <property name="tickInterval">
             <number>1000</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="gainBox">
         <property name="title">
          <string>Gain: 1</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_12">
          <item row="0" column="0">
           <widget class="QSlider" name="gainCoarseSlider">
            <property name="minimum">
             <number>-1000</number>
            </property>
            <property name="maximum">
             <number>1000</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="tickPosition">
             <enum>QSlider::TicksAbove</enum>
            </property>
            <property name="tickInterval">
             <number>1000</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QSlider" name="gainFineSlider">
            <property name="minimum">
             <number>-1000</number>
            </property>
            <property name="maximum">
             <number>1000</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="tickPosition">
             <enum>QSlider::TicksBelow</enum>
            </property>
            <property name="tickInterval">
             <number>1000</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>10</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Calibrate</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Signal Level</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>dB Offset</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="dbOffsetSpinBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="decimals">
             <number>1</number>
            </property>
            <property name="minimum">
             <double>-100.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
            <property name="value">
             <double>-10.500000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Si570 Crystal</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="1" column="0" colspan="2">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Tune and zero beat a precision signal. Enter the exact frequency of your source and press Calibrate. Please allow 10 minutes for the Si570 to warm up before doing this.</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QSpinBox" name="spinBox_2">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimum">
             <number>500000</number>
            </property>
            <property name="maximum">
             <number>40000000</number>
            </property>
            <property name="value">
             <number>15000000</number>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="pushButton_2">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Calibrate</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="2">
           <widget class="QPushButton" name="pushButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Reset to factory default</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>148</width>
           <height>54</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
